graphstorm-processing/graphstorm_processing/data_transformations/dist_feature_transformer.py

```diff
@@ -56,7 +56,9 @@ def __init__(
         if feat_type == "no-op":
             self.transformation = NoopTransformation(**default_kwargs, **args_dict)
         elif feat_type == "numerical":
-            self.transformation = DistNumericalTransformation(**default_kwargs, **args_dict)
+            self.transformation = DistNumericalTransformation(
+                **default_kwargs, **args_dict, json_representation=json_representation
+            )
         elif feat_type == "multi-numerical":
```

Comment:
Can DistMultiNumericalTransformation use the same implementation?

Reply:
We can look into it in another PR, might be possible.